app/test_main.py

```
@@ -1,3 +1,48 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_years, dog_years, expected_result",
+    [
+        (0, 0, [0, 0]),
+        (14, 14, [0, 0]),
+        (0, 4.1, [0, 0]),
+        (14.1, 14, [0, 0]),
```

These test cases for float inputs are in the wrong test function and test for the wrong behavior. According to the task requirements (which specify integer inputs) and checklist item #2.2.3, the `get_human_age` function should raise an exception for non-integer inputs. Please remove these lines and instead add test cases with float values to the `@pytest.mark.parametrize` block of `test_cat_and_dog_years` to verify that a `TypeError` is raised.